package.json

```
@@ -1,31 +1,26 @@
 {
   "name": "@0b5vr/tweakpane-plugin-rotation",
-  "version": "0.1.1",
+  "version": "1.0.0",
```

```suggestion
  "version": "0.2.0",
```

Minor bump would be preferred. Zero versioning.